LongWarOfTheChosen/Config/XComGameData_CharacterStats.ini

```diff
@@ -8867,62 +8867,174 @@ CharacterBaseStats[eStat_FlankingCritChance]=40
 [AdvPurifierM1 X2CharacterTemplate]
 CharacterBaseStats[eStat_ArmorMitigation]=1
 CharacterBaseStats[eStat_HP]=6
-+CharacterBaseStats[eStat_Offense]=160
+CharacterBaseStats[eStat_Offense]=65
+CharacterBaseStats[eStat_Mobility]=10
+CharacterBaseStats[eStat_Will]=55
+
 
 [AdvPurifierM1_Diff_0 X2CharacterTemplate]
 CharacterBaseStats[eStat_HP]=5
-+CharacterBaseStats[eStat_Offense]=150
+CharacterBaseStats[eStat_Offense]=65
+CharacterBaseStats[eStat_ArmorMitigation]=1
+CharacterBaseStats[eStat_Will]=50
 
 [AdvPurifierM1_Diff_2 X2CharacterTemplate]
-CharacterBaseStats[eStat_ArmorMitigation]=2
-CharacterBaseStats[eStat_HP]=7
-+CharacterBaseStats[eStat_Offense]=170
+CharacterBaseStats[eStat_ArmorMitigation]=1
+CharacterBaseStats[eStat_HP]=6
+CharacterBaseStats[eStat_Offense]=70
+CharacterBaseStats[eStat_Will]=55
 
 [AdvPurifierM1_Diff_3 X2CharacterTemplate]
 CharacterBaseStats[eStat_ArmorMitigation]=2
-CharacterBaseStats[eStat_HP]=7
-+CharacterBaseStats[eStat_Offense]=180
+CharacterBaseStats[eStat_HP]=6
+CharacterBaseStats[eStat_Offense]=70
+CharacterBaseStats[eStat_Will]=55
 
 
 [AdvPurifierM2 X2CharacterTemplate]
 CharacterBaseStats[eStat_ArmorMitigation]=2
-CharacterBaseStats[eStat_HP]=9
-+CharacterBaseStats[eStat_Offense]=160
+CharacterBaseStats[eStat_HP]=8
+CharacterBaseStats[eStat_Offense]=70
+CharacterBaseStats[eStat_Mobility]=10
+CharacterBaseStats[eStat_Will]=75
 
 [AdvPurifierM2_Diff_0 X2CharacterTemplate]
 CharacterBaseStats[eStat_HP]=7
-+CharacterBaseStats[eStat_Offense]=150
+CharacterBaseStats[eStat_Offense]=65
+CharacterBaseStats[eStat_ArmorMitigation]=1
+CharacterBaseStats[eStat_Will]=70
 
 [AdvPurifierM2_Diff_2 X2CharacterTemplate]
 CharacterBaseStats[eStat_ArmorMitigation]=3
-CharacterBaseStats[eStat_HP]=10
-+CharacterBaseStats[eStat_Offense]=170
+CharacterBaseStats[eStat_HP]=9
++CharacterBaseStats[eStat_Offense]=70
+CharacterBaseStats[eStat_Will]=75
 
 [AdvPurifierM2_Diff_3 X2CharacterTemplate]
 CharacterBaseStats[eStat_ArmorMitigation]=3
 CharacterBaseStats[eStat_HP]=10
-+CharacterBaseStats[eStat_Offense]=180
-
+CharacterBaseStats[eStat_Offense]=75
+CharacterBaseStats[eStat_Will]=75
 
 [AdvPurifierM3 X2CharacterTemplate]
 CharacterBaseStats[eStat_ArmorMitigation]=3
 CharacterBaseStats[eStat_HP]=11
-+CharacterBaseStats[eStat_Offense]=160
+CharacterBaseStats[eStat_Offense]=75
+CharacterBaseStats[eStat_Mobility]=10
+CharacterBaseStats[eStat_Will]=95
 
 [AdvPurifierM3_Diff_0 X2CharacterTemplate]
 CharacterBaseStats[eStat_ArmorMitigation]=2
-CharacterBaseStats[eStat_HP]=11
-+CharacterBaseStats[eStat_Offense]=150
+CharacterBaseStats[eStat_HP]=10
+CharacterBaseStats[eStat_Offense]=70
+CharacterBaseStats[eStat_Will]=90
 
 [AdvPurifierM3_Diff_2 X2CharacterTemplate]
 CharacterBaseStats[eStat_ArmorMitigation]=4
 CharacterBaseStats[eStat_HP]=13
-+CharacterBaseStats[eStat_Offense]=170
+CharacterBaseStats[eStat_Offense]=75
+CharacterBaseStats[eStat_Will]=95
 
 [AdvPurifierM3_Diff_3 X2CharacterTemplate]
 CharacterBaseStats[eStat_ArmorMitigation]=5
```

Comment:
Does this still need to be so high? Legend typically has the same armor values on units as Commander as far as I'm aware.

Reply:
The purifier will be often run in the open, so I want to still keep them very tanky.